### PR TITLE
fix: web build

### PR DESCRIPTION
**Pull request by mathe42**

### CODE REVIEW

**Comment by oberbeck** [comment]
Should we add a test running the web build in a browser environment?

**Comment by mathe42** [comment]
I don't think that it is worth adding this. We might rerun the deno tests on this build (should work)...

**Comment by tobiemh** [comment]
Is this ready to merge @mathe42?

**Comment by mathe42** [comment]
Yes ready to merge

**Comment by mathe42** [comment]
@tobiemh ready to merge!

File: compile.ts [hunk 1]
```diff
@@ -36,8 +36,9 @@ await build({
 		},
 		scripts: {
 			"build:web":
-				"esbuild ./esm/index.js --bundle --sourcemap --outfile=./web/index.js",
+				"esbuild ./esm/index.js --format=esm --minify --bundle --sourcemap --outfile=./web/index.js",
 		},
+		"browser": "./web/index.js",
```
Comment (kearfy) [inline compile.ts:41]:
```suggestion
		browser: "./web/index.js",
```

**Review by kearfy — APPROVED** [review]

**Pull request merged**